tls/extensions/s2n_server_certificate_status.c

```diff
@@ -23,64 +23,96 @@
 
 #define U24_SIZE 3
 
-static int s2n_server_certificate_status_send_size(struct s2n_connection *conn)
+/* In TLS 1.3, a response to a Status Request extension is sent as an extension with
+ * status request as well as the OCSP response. This contrasts to TLS 1.2 where
+ * the OCSP response is sent in the Certificate Status handshake message */
+
+static bool s2n_tls13_server_status_request_should_send(struct s2n_connection *conn);
+
+const s2n_extension_type s2n_tls13_server_status_request_extension = {
+    .iana_value = TLS_EXTENSION_STATUS_REQUEST,
+    .is_response = true,
+    .send = s2n_server_certificate_status_send,
+    .recv = s2n_server_certificate_status_recv,
+    .should_send = s2n_tls13_server_status_request_should_send,
+    .if_missing = s2n_extension_noop_if_missing,
+};
+
+static bool s2n_tls13_server_status_request_should_send(struct s2n_connection *conn)
+{
+    return s2n_server_can_send_ocsp(conn);
```

Comment:
should this be s2n_server_**should**_send_ocsp? same for s2n_server_can_send_sct_list

Reply:
I'd rather not update the original macro names. They don't necessarily imply that an extension will be sent, just that it can.